tests/test_bad_replica.py

```diff
@@ -249,11 +249,6 @@ def test_client_add_list_bad_replicas(rse_factory, replica_client, did_client):
                     nbbadrep += 1
     assert len(replicas) == nbbadrep
 
-    # InvalidType is raised if list_rep contains a mixture of replicas and PFNs
```

Comment:
On a second look, why was this test removed? It still seems to be raised correctly in the API?

Reply:
If you are talking about test_bad_replica.py lines 253-256, somehow I was not able to make the `with pytest.raises(InvalidType)` catch and mask the exception. So I decided to simply remove that part of the test.

Reply:
This should be fixed in the code then. This is still a valid tests and points to a problem in the code.